src/main/java/main/java/Tasklist.java

```diff
@@ -0,0 +1,107 @@
+package main.java;
+
+import java.util.ArrayList;
+
+public class Tasklist {
+    /* The list that contains all the task objects */
+    protected static ArrayList<Task> list;
+
+    public Tasklist(ArrayList<Task> list) {
+        this.list = list;
+    }
+
+    public ArrayList<Task> getter() {
+        return list;
+    }
+
+    /**
+     * Returns that number of tasks in the list
+     * 
+     * @return the number of tasks
+     */
+    public int size() {
+        return list.size();
+    }
+
+    /**
+     * Adds the task to the list
+     * 
+     * @param task the task the user wants to add
+     */
+    public void addTask(Task task) {
+        list.add(task);
+    }
+
+    /**
+     * Removes the task from the list
+     * 
+     * @param no the number of the task that the user want to remove
+     */
+
+    public void removeTask(int no) {
```

Comment:
Small nitpick is the choice of the parameter "no"; perhaps "number" would be more appropriate for an audience who might not understand the short form notation, such as a non-native English speaker. 
```suggestion
    public void removeTask(int number) {
```